### ISSUE TITLE
Bug: Console messages fails to parse when we have string placeholders *AND* literal `%`

### ISSUE THREAD

**Issue by billyvg** [getsentry/sentry#35637]
See https://sentry.io/organizations/sentry-emerging-tech/replays/new-payload:e6df295b13ae46429e3e18b629e3d403/

**Comment by danecando** [comment]
We should be able to add an aria-current attribute to message items and then re-use the useCurrentItemScroller hook that the breadcrumb scrolling is using. 

I want to wait until #35048 is ready to add this since useCurrentItemScroller is currently using scrollIntoView which might cause some unwanted window scrolling between the console and breadcrumbs lists if they both have a scroller active. Ryan mentioned that the new layout will be a fixed 100vh so we won't have to worry about it. Otherwise we will probably want to refactor useCurrentItemScroller to use scrollTop instead of scrollIntoView so the window doesn't get scrolled automatically

**Comment by ryan953** [comment]
See JAVASCRIPT-2AKF

**Comment by billyvg** [comment]
@ryan953 Oops, not sure if you've started but I made https://github.com/getsentry/sentry/pull/45167

**Comment by billyvg** [comment]
Closed by https://github.com/getsentry/sentry/pull/45167
